apps/date-component/README.md

```diff
@@ -0,0 +1,82 @@
+# Date component example
+
+This tutorial explains the steps to create a date using the [hof-component-date](https://github.com/UKHomeOfficeForms/hof-component-date). There is also a further explanation of useful date validations.
+
+## Getting started
+
+Add a step that displays a date of birth and a field called date-of-birth.
+
+```
+  steps: {
+    '/date-of-birth': {
+      fields: ['date-of-birth'],
+      next: '/confirm'
+    }
+  }
+```
+
+## Adding the Date-component field
+
+Install the [hof-component-date](https://www.npmjs.com/package/hof-component-date) module
+
+Next, go into the fields directory and open the `index.js` file. Add the field with the key `date-of-birth`. Give this key a property of a `dateComponent()`. The date-component is a function that takes arguments.  Give it the arguments of the name of the field `date-of-birth` and an object validate key with an array of properties `['required']`
```

Comment:
I don't think "Give this key a property of a `dateComponent()`" really makes sense. And "is a function that takes arguments" is something of a tautology.

I'd replace this whole paragraph with:

> In your field configuration file, create an instance of a data component. You can do this by calling the data component function with a field name and the options for the field. In this case, our field name is `date-of-birth` and in our options we set a `required` validator.

> Then assign this instance to the appropriate key on your fields config.